packages/blade/src/components/Link/BaseLink/__tests__/BaseLink.web.test.tsx

```diff
@@ -391,4 +392,27 @@ describe('<BaseLink />', () => {
     );
     expect(getByTestId('base-link-test')).toBeTruthy();
   });
+
+  it('should handle styles for all interactions', () => {
+    const textContent = 'Slash';
+    const { getByText } = renderWithTheme(<BaseLink variant="button">{textContent}</BaseLink>);
+
+    // default state
+    const linkButtonText = getByText(textContent);
+    expect(linkButtonText).toHaveStyle({
+      color: paymentTheme.colors.onLight.action.text.link.default,
+    });
```

Comment:
snapshots might be better here?

Reply:
Wanted to add an explicit regression check because it's easy to miss these in snapshots otherwise, we already have lot of snapshots in this component.